spec/features/staff/users_can_create_an_activity_spec.rb

```diff
@@ -0,0 +1,35 @@
+RSpec.feature "Users can create an activity" do
```

Comment:
The activity form is the core function of the service so I think it is worth thoroughly testing. We can add a missing scenario for validating the ID and fund/parent, I think it would be a good idea to test the validation errors appear and then correct the errors and resubmit in the same test. That would have caught an error I observed when testing where the fund value gets set wrong and prevents the form being resubmitted when the ID is omitted.

Reply:
As I've changed the routes so that you can't get to the Activity#new view without a fund ID, I think this is OK now - but please check!

Reply:
We've removed the fund ID which is good and removes this unexpected scenario.

As well as fixing it, I was getting at extending our test coverage a little so that we catch any future regressions. Given that we no longer have a validations for this so that we can do the following, it can be added later:

```
1. fill out form with errors
2. submit invalid form
3. assert that validations are shown
4. correct errors
5. submit form again
6. assert that everything works
```